OSDOCS-13985: updates multus in config.yaml MicroShift (#91783)
openshift/openshift-docs: modules/microshift-install-multus-rpm.adoc

[id="microshift-installing-multus_{context}"]
= Installing the multiple networks plugin

> Use this procedure to install the {microshift-short} Multus Container Network Interface (CNI) plugin alongside a new {microshift-short} installation. If you want to attach additional networks to a pod for high-performance network configurations, install the `microshift-multus` RPM package.

Reviewer: To avoid self-referential language:
```suggestion
You can install the {microshift-short} Multus Container Network Interface (CNI) plugin alongside a new {microshift-short} installation. If you want to attach additional networks to a pod for high-performance network configurations, install the `microshift-multus` RPM package.
```

[IMPORTANT]
====

> The {microshift-short} Multus CNI plugin is installed by default. Alternatively, you can set the value in the {microshift-short} `config.yaml` file to `Enabled`. Uninstalling the {microshift-short} Multus CNI is not supported in either case.

Reviewer 2: Similar to relnote PR:
We don't install multus by default. Maybe 'install' is a wrong word here.
MicroShift embeds the manifests, but deploys them only after .network.multus.status is set to Enabled.

Summary for me is:
Multus manifests are now part of MicroShift binary. You can install RPM to enable it or enable it via config. After enabling (whatever the method, they are the same in the end), it cannot be disabled (or rather disabling it won't remove the cluster objects).
